DERMS/Common/WeatherAPI/WeatherApiClient.cs

```diff
@@ -18,9 +12,8 @@ public class WeatherApiClient : IWeatherClient
         private static readonly string cityParameter = "@CITY@";
 
         private static readonly string forecastParameter = "forecast";
-        private static readonly string historyParameter = "history";
 
-        private static readonly int futureDays = 3;
+        private static readonly int futureDays = 2;
```

Comment:
Meni je citljivije da nazivi svih ovih konstanti budu komplet velikim slovima kako bi ih razlikovali od obicnih promenljivih.

Reply:
Uglavnom se property pise velikim slovima a polja malim, opet neki standard na koji sam naucio, mozemo glasanjem da odlucimo kako cemo.

Reply:
```suggestion
        private static readonly int FUTURE_DAYS = 2;
```

Ovako sam mislio. Ni property ni obicna promenljiva :)

Reply:
Jasno, slazem se.